mem/vm/mmu/interface.go

```diff
@@ -0,0 +1,7 @@
+package mmu
+
+// pageSizeGetter is an optional interface that page tables can implement
+// to expose their page size for validation purposes.
+type pageSizeGetter interface {
```

Comment:
Update this interface to `pageTable`. Aggregate all the methods of the page table that is used in the MMU package.